LabDrivers/__init__.py

```diff
@@ -1,2 +1,2 @@
-__all__ = ['A34401A', 'AH2550A', 'ArduinoUno', 'CG500', 'DAC488', 'E4400B', 'FridgeClient', 'GDS820C', 'HLT560', 'HP34401A', 'HP4263B', 'IPS120', 'KT2400',
+__all__ = ['A34401A', 'AH2550A_legacy.py', 'ArduinoUno', 'CG500', 'DAC488', 'E4400B', 'FridgeClient', 'GDS820C', 'HLT560', 'HP34401A', 'HP4263B', 'IPS120', 'KT2400',
```

Comment:
Why do you need to rename it with legacy? If the driver is updated we don't need a legacy, do we? If people want the old code back, they can checkout and older branch.

Reply:
I believe this was automatically generated via refactor with PyCharm when I rewrote the AH2550A driver. Thank you for bringing this to my attention I will fix it right away